<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Playlist</class>
 <widget class="QWidget" name="Playlist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Playlist</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_vertical_01">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QListWidget" name="listWidget_playlist">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::InternalMove</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_iteminfo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>744</width>
         <height>68</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_iteminfo">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>label_iteminfo</zorder>
       <zorder>splitter</zorder>
       <zorder>label_artwork</zorder>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolButton" name="toolButton_moveup">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_movedown"/>
     </item>
     <item>
      <widget class="Line" name="line_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_add"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_remove"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_removeall"/>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_exit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="2">
      <widget class="QLabel" name="label_artwork">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="checkBox_wrap">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;At the last item wrap to the beginning&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>&amp;Wrap</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_consume">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remove items from list after playing&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>&amp;Consume</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_random">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Play entries in a random order&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>&amp;Random</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_showinfo">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Show additional details about the selected item including album art if available.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>&amp;Details</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_summary">
         <property name="text">
          <string>Summary</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_save">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionMoveUp">
   <property name="text">
    <string>Move Up</string>
   </property>
   <property name="toolTip">
    <string>Move Selected Item Up</string>
   </property>
  </action>
  <action name="actionMoveDown">
   <property name="text">
    <string>Move Down</string>
   </property>
   <property name="toolTip">
    <string>Move Selectes Item Down</string>
   </property>
  </action>
  <action name="actionAddMedia">
   <property name="text">
    <string>Add Media</string>
   </property>
  </action>
  <action name="actionRemoveItem">
   <property name="text">
    <string>Remove Item</string>
   </property>
   <property name="toolTip">
    <string>Remove Selected Item</string>
   </property>
  </action>
  <action name="actionHidePlaylist">
   <property name="text">
    <string>Hide Playlist</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionAddFiles">
   <property name="text">
    <string>Add Files</string>
   </property>
   <property name="toolTip">
    <string>Add Files</string>
   </property>
  </action>
  <action name="actionAddURL">
   <property name="text">
    <string>Add URL</string>
   </property>
  </action>
  <action name="actionAddAudio">
   <property name="text">
    <string>Add Audio</string>
   </property>
   <property name="toolTip">
    <string>Add Audio File</string>
   </property>
  </action>
  <action name="actionAddVideo">
   <property name="text">
    <string>Add Video</string>
   </property>
   <property name="toolTip">
    <string>Add Video File</string>
   </property>
  </action>
  <action name="actionRemoveAll">
   <property name="text">
    <string>Remove All</string>
   </property>
   <property name="toolTip">
    <string>Remove All Items</string>
   </property>
  </action>
  <action name="actionSavePlaylist">
   <property name="text">
    <string>Save Playlist</string>
   </property>
   <property name="toolTip">
    <string>Save the playlist to disk.</string>
   </property>
  </action>
  <action name="actionAddPlaylist">
   <property name="text">
    <string>Add Playlist</string>
   </property>
   <property name="toolTip">
    <string>Add Files from .m3u playlist</string>
   </property>
  </action>
  <action name="actionToggleWrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wrap</string>
   </property>
  </action>
  <action name="actionToggleConsume">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Consume</string>
   </property>
  </action>
  <action name="actionToggleRandom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Random</string>
   </property>
  </action>
  <action name="actionToggleDetail">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toggle Detail</string>
   </property>
   <property name="toolTip">
    <string>Detail</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../mbmp.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkBox_consume</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBox_wrap</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>49</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_wrap</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBox_consume</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_showinfo</sender>
   <signal>toggled(bool)</signal>
   <receiver>scrollArea_iteminfo</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
